src/mono/wasm/runtime/library_mono.js

```diff
@@ -1301,15 +1301,15 @@ var MonoSupportLib = {
 					try {
 						if (asset.name === attemptUrl) {
 							if (ctx.tracing)
-								console.log ("Attempting to fetch '" + attemptUrl + "'");
+								console.log ("Attempting to fetch '%s'", + attemptUrl);
```

Comment:
this will convert attemptUrl into a double. Also, the %s doesn't seem like it should work? Use backtick literals

Reply:
nvm on the ```%s``` thing, while it's not documented in most places I found somewhere that does document it and it appears to work in firefox and chrome, so it's fine not to use a backtick literal if you want